### PR TITLE
Fix decodeRawTransaction network addresses

### PR DESCRIPTION
**Pull request by matthewjablack**
### Description

This PR fixes decodeRawTransaction network addresses. 

### Submission Checklist :pencil:

- [x] Ensure `decodeRawTransaction` is passed network

### CODE REVIEW

File: packages/bitcoin-wallet-provider/lib/BitcoinWalletProvider.js [hunk 1]
```diff
@@ -38,9 +38,10 @@ export default superclass => class BitcoinWalletProvider extends superclass {
   }
 
   async _sendTransaction (transactions, feePerByte) {
+    const network = this._network
     const { hex, fee } = await this._buildTransaction(transactions, feePerByte)
     await this.getMethod('sendRawTransaction')(hex)
-    return normalizeTransactionObject(decodeRawTransaction(hex), fee)
+    return normalizeTransactionObject(decodeRawTransaction(hex, network), fee)
```
Comment (monokh) [inline packages/bitcoin-wallet-provider/lib/BitcoinWalletProvider.js]: would be clearer to just pass `this._network` here and avoid the reassignment

File: packages/bitcoin-wallet-provider/lib/BitcoinWalletProvider.js [hunk 2]
```diff
@@ -56,12 +57,14 @@ export default superclass => class BitcoinWalletProvider extends superclass {
   }
 
   async sendSweepTransaction (externalChangeAddress, feePerByte, outputs, fixedInputs) {
+    const network = this._network
```
Comment (monokh) [inline packages/bitcoin-wallet-provider/lib/BitcoinWalletProvider.js]: ditto

File: packages/bitcoin-wallet-provider/lib/BitcoinWalletProvider.js [hunk 3]
```diff
@@ -56,12 +57,14 @@ export default superclass => class BitcoinWalletProvider extends superclass {
...
     const { hex, fee } = await this._buildSweepTransaction(externalChangeAddress, feePerByte, outputs, fixedInputs)
     await this.getMethod('sendRawTransaction')(hex)
-    return normalizeTransactionObject(decodeRawTransaction(hex), fee)
+    return normalizeTransactionObject(decodeRawTransaction(hex, network), fee)
   }
 
   async updateTransactionFee (tx, newFeePerByte) {
+    const network = this._network
```
Comment (monokh) [inline packages/bitcoin-wallet-provider/lib/BitcoinWalletProvider.js]: ditto

File: packages/bitcoin-wallet-provider/lib/BitcoinWalletProvider.js [hunk 4]
```diff
@@ -237,6 +240,7 @@ export default superclass => class BitcoinWalletProvider extends superclass {
   }
 
   async getInputsForAmount (_targets = [], _feePerByte, fixedInputs = [], numAddressPerCall = 100, sweep = false) {
+    const network = this._network
```
Comment (monokh) [inline packages/bitcoin-wallet-provider/lib/BitcoinWalletProvider.js]: ditto

**Review by monokh — APPROVED** [review]
LGTM, could you change target to dev?

**Comment by matthewjablack** [comment]
Good point @monokh. Updated to use `this._network` to avoid reassignment

**Pull request merged**